backend/app/controllers/api/v3/permissions_controller.rb

```diff
@@ -2,14 +2,17 @@
 
 module Api
   module V3
-    # PERMISSIONS RELATED API CALLS
+    # Permissions related api calls
     class PermissionsController < ApplicationController
-      # GET CURRENT LIST OF PERMISSIONS FROM PERMISSIONS DB
-      def get_permissions
+      # Returns the list of permissions.
+      #
+      # @param token [String] a token
+      #
+      # @return the list of permissions
+      def index
+         # Check for any permissions
         status, response = check_token_for_permission
-        if response[:error]
```

Comment:
this is an example of a spot where it looks like you didn't merge the v3-refactor, but I understand you did do a merge.
What appears to have happened is that your merge resolved conflicts in favor of your code and so anything different in backend in v3-refactor was replaced.